Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -42,10 +43,10 @@ configuration:
 - additionalinfo: Whether to send or not automatic notifications to the configured notifications channel.
   display: Automatic Notifications
   hidden: false
-  name: auto_notifications
+  name: disable_auto_notifications
```

Comment:
cant we just keep the same ?

Reply:
We can. Thought it might be confusing for the reader, wdyt?

Reply:
the user doesnt actually see the param name, just the display

Reply:
Yeah, meant confusing as for the code reader, not user. But i'll revert to auto_notifications